scripts/pi-hole/js/db_graph.js

```diff
@@ -71,7 +72,35 @@ function compareNumbers(a, b) {
 function updateQueriesOverTime() {
     $("#queries-over-time .overlay").show();
     timeoutWarning.show();
-    $.getJSON("api_db.php?getGraphData&from="+from+"&until="+until, function(data) {
+
+    // Compute interval to obtain about 200 values
+    var num = 200;
```

Comment:
This variable controls how many lines will be computed. I don't think we need to make it user-defined, however, I put it in a common place so users can still modify it when they deem it appropriate. We could also make it a global variable, users could then modify this value (live) through the developer tools.